frontend/src/components/pod/List.tsx

```diff
@@ -22,7 +22,7 @@ export function makePodStatusLabel(pod: Pod) {
     status = 'error';
   } else if (phase === 'Succeeded' || phase === 'Running') {
     const readyCondition = pod.status.conditions.find(condition => condition.type === 'Ready');
-    if (readyCondition?.status === 'True') {
+    if (readyCondition?.status === 'True' || phase === 'Succeeded') {
```

Comment:
This makes me think should we add a blue color for all the succeeded ones because now running and succeeded have the same color

Reply:
Let's worry about that later. I think Running and Succeeded can be green for now. But it's a good remark.